platform-sdk/docs/proposals/template.md

```diff
@@ -0,0 +1,119 @@
+# Summary
+
+Give a 1-3 sentence summary of the capability in the design proposal.
+
+| Metadata           | Entities                            | 
+|--------------------|-------------------------------------|
+| Last Updated       | 2024-06-07  (Edward)                |
+| Designers          | John, Mary, alpha-team              |
+| Functional Impacts | Services, DevOps, Mirror Node, etc. |
+| Related Proposals  | Proposal-1, Proposal-2              |
+| HIPS               | HIP-1, HIP-2,                       |
+
+# Purpose and Context
+
+Why is the design proposal necessary? Describe the background and purpose of the design proposal at a high level.
+
+## Dependencies, Interactions, and Implications
+
+What are the dependencies and pre-requisites of this proposal?  
+What other proposals, capabilities, and teams are impacted by this proposal?
+What does the proposal enable once it is completed?
+
+Remove this section if not applicable.
+
+# Requirements
+
+Describe the requirements and acceptance criteria that the design proposal must satisfy.
+
+# Design Decisions
+
+Describe the high level options available, the trade-offs between them and the decisions made.
```

Comment:
I think this needs more explanation. This is also two sections, in my opinion. The decisions made, and reason why is one. The other is design alternates and why they were not chosen. The notion doc also mentions a design table with criteria and scores.

Reply:
I've modified this section to reflect your suggested changes.